src/Diagnostics/variables.jl

```diff
@@ -541,4 +541,7 @@ function setup_variables()
         "vort2",
         var_attrib("s^-1", "vorticity from DG kernels", ""),
     )
+    Variables["hyper_e"] = DiagnosticVariable("hyper_e", var_attrib("", "", ""))
+    Variables["hyper_u"] = DiagnosticVariable("hyper_u", var_attrib("", "", ""))
+    Variables["hyper_v"] = DiagnosticVariable("hyper_v", var_attrib("", "", ""))
```

Comment:
@jiahe23: please fill in these definitions.

Reply:
Done!